crates/sui-core/src/authority/shared_object_congestion_tracker.rs

```diff
@@ -346,7 +347,7 @@ mod object_cost_tests {
             10,
         ) {
             assert_eq!(future_round, 11);
-            assert_eq!(deferred_from_round, 10);
+            assert_eq!(deferred_from_round, 5);
```

Comment:
Consider changing the `deferred_from_round` to something other than 5, so that it is different from the below test case